recipes/unitig-counter/build.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# set up directory for building boost compiled dependencies
+# see
+# https://github.com/conda-forge/boost-cpp-feedstock/blob/master/recipe/build.sh
+# https://github.com/boostorg/program_options/blob/develop/.travis.yml
+git clone --depth 1 --single-branch --branch boost-1.69.0 https://github.com/boostorg/boost.git boost
```

Comment:
Can't you use boost from conda-forge?

Reply:
Yes, this does work with this one! Changed to do that